pilot/pkg/xds/eds_sh_test.go

```diff
@@ -296,3 +337,12 @@ func addNetwork(server *xds.FakeDiscoveryServer, id network.ID, network *meshcon
 	meshNetworks.Networks = c
 	server.Env().SetNetworks(&meshNetworks)
 }
+
+func getLbEndpointAddrs(eps []*endpoint.LbEndpoint) []string {
+	addrs := make([]string, 0)
+	for _, lbEp := range eps {
+		addrs = append(addrs, lbEp.GetEndpoint().Address.GetSocketAddress().Address)
+	}
+	sort.Strings(addrs)
```

Comment:
Discussed offline. Decided that this sort should remain, so that the return values are deterministic. The eds test has a sort as well, which was the main concern here, but it's only used for formatting for the error strings - not validation.